Adds X server info to run within Docker (#147)
aguirre-lab/ml4c3-wiki: Docker.md

docker push ghcr.io/aguirre-lab/ml4c3:latest-cpu
docker push ghcr.io/aguirre-lab/ml4c3:latest-gpu
```

### Run Docker with X server

X server is a windowing system for bitmap displays, common on linux operating systems. If a recipe is run in Docker and needs to display a figure
or render an animation, X server will allow doing so.

> On the Docker side, X server is already configured and adding the argument `-x` you can run the recipe with it.

Reviewer: ```suggestion
On the Docker side, X server is already configured and enabled by using the `-x` argument with `run.sh`.
```

Nevertheless, each workstation must be set up to run ssh with X server as well. The following steps should be followed to properly configure it:
1. Install xauth on the workstation (most likely it will be already installed): 
```
sudo apt-get install -y xauth
```
2. Set default configuration on ssh. Go to `/etc/ssh/sshd_config` and uncomment `X11Forwarding yes`.
3. Ensure that you have `.Xauthority` file on your home directory. Installing `xauth` create it automatically. If you can't find it, try to reinstall 
`xauth` or restart `gdm3`:
```
sudo apt-get --purge remove xauth
sudo apt-get install -y xauth

sudo systemctl restart gdm3
sudo service gdm start
```

> 4. Logout from ssh and connect again with `-X` argument.

Reviewer: The last step is that X server needs to be installed on the remote machine, on windows Xming, on mac Xquartz

because it goes: remote --> workstation --> docker and each one of those needs X server

sudo systemctl restart gdm3
sudo service gdm start
```
- Install X server on the remote machine. For Windows, install Xming ([here](https://sourceforge.net/projects/xming/) you can download it). 
Also, take into account you will need [PuTTY](www.putty.org) to run X server and use Xming. Instructions on how to set up Xming and Putty are explained
on this [page](http://laptops.eng.uci.edu/software-installation/using-linux/how-to-configure-xming-putty). For macOS, install Xquartz ([here](https://www.xquartz.org/) you can download it)

> Instructions on how to set up Xquarts are explained on this [page](https://content.byui.edu/file/cddfb9c0-a825-4cfe-9858-28d5b4c218fe/1/Course/Setup-XQuartz.html).

Reviewer: ```suggestion
Instructions on how to set up Xquartz are explained on this [page](https://content.byui.edu/file/cddfb9c0-a825-4cfe-9858-28d5b4c218fe/1/Course/Setup-XQuartz.html).
```
